kore/test/Test/Kore/Repl/Parser.hs

```diff
@@ -222,11 +226,12 @@ pipeTests =
 
 pipeRedirectTests :: [ParserTest ReplCommand]
 pipeRedirectTests =
-    [ "config | script > file"                     `parsesTo_` pipeRedirectConfig Nothing "script" [] "file"
-    , "config 5 | script \"a r g 1\" arg2 > file"  `parsesTo_` pipeRedirectConfig (Just 5) "script" ["a r g 1", "arg2"] "file"
-    , "config 5 | > "                              `fails`     "no script or file name"
-    , "config 5 | script > "                       `fails`     "no file name"
-    , "config 5 | > file"                          `fails`     "no script name"
+    [ "config | script > file"                          `parsesTo_` pipeRedirectConfig Nothing "script" [] "file"
```

Comment:
These doesn't look like it fits 80 characters, but I don't have a good idea how to make this look good. @marick any suggestions?

Reply:
Break each line before the operator?